### ISSUE TITLE
Update deps: part 2

### ISSUE THREAD

**Issue by danuker** [chevah/github-hooks-server#69]
Revenge of #67 

`grpcio` was obsolete because `azure-functions-worker` was obsolete. Trying the [official MS guide](https://learn.microsoft.com/en-us/azure/azure-functions/create-first-function-cli-python?tabs=linux%2Cbash%2Cazure-cli%2Cbrowser#create-a-local-function), the requirements only include `azure-functions`, and nothing else.

See if the deployment works without `azure-functions-worker`.

**Comment by adiroiban** [comment]
We can just remove it from here 

https://github.com/chevah/github-hooks-server/blob/3e956426ba850371fd0c2256cadedec108bf9a3d/requirements.txt#L217

------

We might want to have 2 requirement files.

The high-level `requirements.txt` with only the direct deps, and without hash.

And a `Pipfile.lock` with the full list of deps and hash values

**Comment by danuker** [comment]
`requirements.txt` was created automatically by `poetry-plugin-export` from the `[tool.poetry.dependencies]` section of `pyproject.toml`.

I did this to avoid having to manually handle tons of dependency hashes which Azure Functions requires.

I also use `requirements.txt` as a witness that the project worked with those versions (so, the same role as `Pipfile.lock`). Poetry also creates `poetry.lock` but it would be redundant to commit it to the repo.

---

It seems we don't need `azure-functions-worker`, `grpcio`, `protobuf`, nor Node anymore. We needed Node for Serverless, but it has been replaced by the trivial `build-package.sh`.
